controller/filehandlingcontroller.php

```diff
@@ -146,6 +162,24 @@ public function save($path, $filecontents, $mtime) {
 				} else {
 					// File same as when opened, save file
 					if ($this->view->isUpdatable($path)) {
+						$fileInfo = $this->view->getFileInfo($path);
+						$storage = $fileInfo->getStorage();
+						$lookingEnabled = $this->config->getAppValue("files", "enable_lock_file_action", "no") === "yes";
```

Comment:
you should check for lock even if the app setting is disbled.
the app config only controlls if the front end for locking is enabled.

files can be locked even if the config is off